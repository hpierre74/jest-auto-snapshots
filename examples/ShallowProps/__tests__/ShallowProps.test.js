import snap from 'jest-auto-snapshots';
import ShallowProps from '../ShallowProps';

snap(ShallowProps);
