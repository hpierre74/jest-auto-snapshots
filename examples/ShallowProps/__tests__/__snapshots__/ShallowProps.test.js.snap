// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest-auto-snapshots > ShallowProps Matches snapshot when boolean prop "sayHello" is set to: "false" 1`] = `
<div>
  <h1>
    jest-auto-snapshots String Fixture
  </h1>
  <div>
    <NodeFixture />
  </div>
  <button
    onClick={[MockFunction]}
  >
    button
  </button>
  jest-auto-snapshots String Fixture
  1
  News
</div>
`;

exports[`jest-auto-snapshots > ShallowProps Matches snapshot when passed all props 1`] = `
<div>
  <h1>
    jest-auto-snapshots String Fixture
  </h1>
  <span>
    Hello!
  </span>
  <div>
    <NodeFixture />
  </div>
  <button
    onClick={[MockFunction]}
  >
    button
  </button>
  jest-auto-snapshots String Fixture
  1
  News
</div>
`;

exports[`jest-auto-snapshots > ShallowProps Matches snapshot when passed only required props 1`] = `
<div>
  <h1>
    jest-auto-snapshots String Fixture
  </h1>
  <div>
    <NodeFixture />
  </div>
  <button
    onClick={[MockFunction]}
  >
    button
  </button>
  999
  News
</div>
`;
